### PR TITLE
docs(mac): Add note about installing with homebrew

### PR DESCRIPTION
**Pull request by darcywong00**
Add a note on installing Keyman for macOS with homebrew

@keymanapp-test-bot skip

### CODE REVIEW

File: mac/help/start/install-keyman.md [hunk 1]
```diff
@@ -3,7 +3,12 @@ title: Install Keyman for macOS
 ---
 
 *Keyman for macOS* is available for [download](https://keyman.com/mac/download) as a standard
-Apple Disk Image (.dmg) file. It should be installed according to the following instructions.
+Apple Disk Image (.dmg) file. You can also get the installer through Homebrew with the command:
+```
+brew install --cask keyman
+```
```
Comment (mcdurdin) [inline mac/help/start/install-keyman.md:9]: If installing with Homebrew, then no need to install step-by-step, but will need to do additional security changes (I can't remember the differences off the top of my head).

**Review by mcdurdin — COMMENTED** [review]
I think I'd like to list the primary source first -- that is, our download link -- and then homebrew as an alternative under its own heading. Sorry to be a pain!

**Review by mcdurdin — APPROVED** [review]

File: mac/help/start/install-keyman.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: Install Keyman for macOS
 ---
 
-*Keyman for macOS* is available for [download](https://keyman.com/mac/download) as a standard
+*Keyman for macOS* is available for [downloaded](https://keyman.com/mac/download) as a standard
```
Comment (mcdurdin) [inline mac/help/start/install-keyman.md:5]:
```suggestion
*Keyman for macOS* is available for [download](https://keyman.com/mac/download) as a standard
```

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.122-alpha](https://keyman.com/downloads/releases/alpha/15.0.122)

**Pull request merged**